server/config/oauth2/providers.json

```diff
@@ -0,0 +1,11 @@
+{
+  "test": {
```

Comment:
What is this file? localhost? test?

Reply:
this file contains configuration of oaut2 provider (lire url to api) , I put an entry name test for unit test

Reply:
I remove provider.json to put it in db